contracts/ArrakisV2.sol

```diff
@@ -115,111 +106,75 @@ contract ArrakisV2 is IUniswapV3MintCallback, ArrakisV2Storage {
             token1.safeTransferFrom(msg.sender, me, amount1);
         }
 
-        emit LogUncollectedFees(fee0, fee1);
```

Comment:
why remove this line? can't we grab the fees from `totalUnderlyingWithFees` and collect them on mint? This event was just a helper a.k.a. helps us get a sense on fee growth at every supply change for subgraph and APR tracking. So maybe we could do this another way and don't need this event at all, but if we are still using it in other places it should stick around here too.

Reply:
Ok I see you removed this event entirely. As long as we still have alternate way of tracking this in subgraph, I support the removal